_oasis

```diff
@@ -365,11 +369,19 @@ Library sema_test
   Install:        false
   Modules:        Test_ir
 
+Library serialization_test
+  Path:		  piqi/test
+  Build$:	  flag(tests) && flag(serialization)
+  CompiledObject: best
+  BuildDepends:	  bap, oUnit, bap.serialization
+  Install:	  false
+  Modules:	  Test_serialization
+
 Library benchmarks
   Path:           benchmarks
   Build$:         flag(tests) && flag(benchmarks)
   CompiledObject: best
-  BuildDepends:   bap, core, core_bench, threads
+  BuildDepends:   bap, core_bench, threads
```

Comment:
I was getting warnings it wasn't needed, so I disabled it. It's likely due
to my core_kernel version, I had a to edit because of conflicts with other
packages on my box. It was <=, I made it < only though. I can edit the
commit to drop that if you want.
On Oct 23, 2015 6:27 AM, "Ivan Gotovchits" <notifications@github.com> wrote:

> In _oasis
> <https://github.com/BinaryAnalysisPlatform/bap/pull/307#discussion_r42851467>
> :
>
> >  Library benchmarks
> >    Path:           benchmarks
> >    Build$:         flag(tests) && flag(benchmarks)
> >    CompiledObject: best
> > -  BuildDepends:   bap, core, core_bench, threads
> > +  BuildDepends:   bap, core_bench, threads
>
> as far as I remember, core_bench depends on core. Did they change
> something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BinaryAnalysisPlatform/bap/pull/307/files#r42851467>.
>